### PR TITLE
ID - 39: Order details history page integration

### PR DESCRIPTION
**Pull request by saletG08**
### What?

What I did is that the user who is logged in and tries to go to the "Order history" section will search the database if the user made a purchase, but if they did not make a purchase it will show them a screen where it says : "You didn't buy any product!" with a small paragraph that says "Please go see the products in our store to purchase." Later, at the bottom of this paragraph there will be a button that will guide the user to the products they can buy, which the button will say "See Products" and when clicking on the button it will take them to the products screen.
Upon successfully making the purchase, the user will be able to go to the "Order history" screen again and it will show all the products they purchased in a table. Before the table appears it will be loaded with data so that the data can be retrieved until it loads.
The screen is responsive and the table will get smaller.

### How?
I did it with the help of the API that has to send requests to the OrderHistory which was created and it will return the userId with this userId we will look for the user if they buy a product to show in the table.
For the responsive one, only CSS was used to change the letter sizes, colors, among others.

### Why?
I did it with the purpose that the user can see all the products that he made throughout the navigation, in order to see which products he liked or which he is not interested in. In addition, the objective of this week is to complete the purchase and what we have is the visualization of the products that the user has.

Author: @saletG08 
Code Reviewer: @Elvis-Castro-H  
Tester: @Karvlox

### CODE REVIEW

File: next-env.d.ts
Comment (Elvis-Castro-H) [inline next-env.d.ts:1]: delete this file

**Review by Elvis-Castro-H — CHANGES_REQUESTED** [review]
Resolve comments

File: next-env.d.ts
Comment (saletG08) [reply next-env.d.ts:1]: The file has been deleted

**Review by Karvlox — APPROVED** [review]
Good job

**Comment by victor-villca** [comment]
@saletG08 The UI and functionality of this pull request does not match the one talked on the planning, please fix it.

**Review by Elvis-Castro-H — APPROVED** [review]
Good job

**Pull request merged**